package/MDAnalysis/converters/RDKitParser.py

```diff
@@ -156,6 +152,8 @@ class RDKitParser(TopologyReaderBase):
     .. versionadded:: 2.0.0
     .. versionchanged:: 2.1.0
        Added R/S chirality support
+    .. versionchanged:: 2.4.0
+      removed type guessing (guessing takes place now inside universe)
```

Comment:
The change of behaviour regarding types and elements should appear in the `versionchanged`.